src/env/env.ts

```diff
@@ -0,0 +1,211 @@
+#!/usr/bin/env node
+/* eslint-disable no-process-exit */
+import * as fs from 'fs/promises';
```

Comment:
please move the file to src folder. We don't have specific folders for each script

Reply:
in this case, where the if-env template manifest be placed? 
@jmcook1186 should it be moved to the` ./manifests/examples` folder with the name `if-env-template.yml`?

Reply:
no, it should just be saved to the top level `src` directory. `if-env` is not supposed to be run inside the `if` source repository